src/core/src/Gsim.cc

```diff
@@ -184,7 +184,8 @@ void Gsim::BeginOfRunAction(const G4Run* /*aRun*/) {
     DBLinkPtr lmc = DB::Get()->GetLink("MC");
     runID = DB::Get()->GetDefaultRun();
     utc = TTimeStamp();  // default to now
-    
+    startTime = TTimeStamp(); 
```

Comment:
Any reason that we need both utc and startTime? It seems like the two should always be identical (since utc is how G4 is setting times from what I can tell). I guess eventually this whole section needs to be changed anyways since it really should ask the DS::Run what the time is, not the other way around.

Reply:
You're correct. I've removed 'startTime'.